websites/N/Night.design/presence.ts

```diff
@@ -0,0 +1,116 @@
+const presence = new Presence({
+	clientId: "1031730262092042321",
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "https://i.imgur.com/4mDvKNa.png",
+	};
+
+	presenceData.startTimestamp = Math.floor(Date.now() / 1000);
+
+	const pathname = document.location.pathname.slice(3);
+
+	switch (pathname) {
+		case "/home":
+			presenceData.details = "View the Homepage";
+			break;
+		case "/resources":
+			presenceData.details = "Search...";
+			break;
+		case "/settings/account":
+			presenceData.details = "Change my Account Settings";
+			break;
+		case "/settings/notifications":
+			presenceData.details = "Change my Notifications Settings";
+			break;
+		case "/settings/connected-accounts":
+			presenceData.details = "Add a user Connection to my account";
+			break;
+		case "/settings/account-security":
+			presenceData.details = "Change my Account Security Settings";
+			break;
+		case "/password-forgotten":
+			presenceData.details = "Change my Password";
+			break;
+		case "/login":
+			presenceData.details = "Sign in with my Account";
+			break;
+		case "/register":
+			presenceData.details = "Create a new Account";
+			break;
+		case "/dashboard":
+			presenceData.details = "View my Dashboard";
+			break;
+		case "/creator":
+			presenceData.details = "View Creator page";
+			break;
+		case "/modder":
+			presenceData.details = "View Modder Page";
+			break;
+		case "/modder/publisher":
+			presenceData.details = "Apply to become a Modder";
+			break;
+		case "/legal/terms-of-service":
+			presenceData.details = "Read the Terms of Service from Night.design";
+			break;
+		case "/legal/privacy-policy":
+			presenceData.details = "Read the Privacy Policy from Night.design";
+			break;
+		case "/legal/imprint":
+			presenceData.details = "Read the Imprint from Night.design";
```

Comment:
> if and else-ifs should be put in a Record if they're all static

These are static and should use the Record syntax:
https://github.com/PreMiD/Presences/blob/236523980322a4053a45c691a1739eb50e2aea48/websites/G/Geotastic/presence.ts#L12-L42